charts/gcp-managed-cert/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: gcp-managed-cert
+description: Chart for creating gcp managed certs
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+appVersion: 0.1
```

Comment:
Can we leave this out? It's not required, and it doesn't really mean anything in this context.

Reply:
Or else maybe set it to "networking.gke.io/v1beta1" to match the API version